### PR TITLE
Sample Creator

### PR DESCRIPTION
**Pull request by aarati-K**
WIP. Do not merge.

### CODE REVIEW

File: sampling/batch_creator.py [hunk 1]
```diff
@@ -23,21 +23,28 @@ def run(self):
...
+                print("Waiting for sample creator to create a sample")
+                continue
             else:
                 i = 0
-                if not curr_sample or self.offset == self.ds.sample_size:
-                    self.offset = 0
-                    curr_sample = self.ds.samples.get()
-                curr_batch = []
+                if cur_sample is None:
+                    cur_sample = self.ds.samples.get()
```
Comment (SrujithPoondla) [inline sampling/batch_creator.py:36]: Whenever you are picking a new sample set offset to 0

Comment (aarati-K) [reply sampling/batch_creator.py:36]: I'm setting it in line 47.

Comment (SrujithPoondla) [reply sampling/batch_creator.py:36]: is there any improvement with this change or was there any error?

Comment (SrujithPoondla) [reply sampling/batch_creator.py:36]: Is there any improvement with this change or was there any error?

Comment (aarati-K) [reply sampling/batch_creator.py:36]: There was an error. I will explain in person.

Comment (SrujithPoondla) [reply sampling/batch_creator.py:36]: okay

**Comment by aarati-K** [comment]
Merging this.

**Pull request merged**